ddtrace/contrib/aiomysql/patch.py

```diff
@@ -0,0 +1,118 @@
+import asyncio
+
+import aiomysql
+
+from ddtrace import Pin
+from ddtrace import config
+from ddtrace.constants import ANALYTICS_SAMPLE_RATE_KEY
+from ddtrace.constants import SPAN_MEASURED_KEY
+from ddtrace.contrib import dbapi
+from ddtrace.ext import sql
+from ddtrace.internal.utils.wrappers import unwrap
+from ddtrace.vendor import wrapt
+
+from ...ext import SpanTypes
+from ...ext import db
+from ...ext import net
+
+
+config._add(
+    "aiomysql",
+    dict(_default_service="mysql"),
+)
+
+CONN_ATTR_BY_TAG = {
+    net.TARGET_HOST: "host",
+    net.TARGET_PORT: "port",
+    db.USER: "user",
+    db.NAME: "db",
+}
+
+
+async def patched_connect(connect_func, _, args, kwargs):
+    conn = await connect_func(*args, **kwargs)
+    tags = {t: getattr(conn, a, "") for t, a in CONN_ATTR_BY_TAG.items()}
```

Comment:
```suggestion
    tags = {}
    for tag, attr in CONN_ATTR_BY_TAG.items():
        if hasattr(conn, attr):
            tags[tag] = getattr(conn, attr)
```

this is not a strong/required suggestion. my thinking here is what if we end up have a tag as `""`, would it be better to just omit entirely? what would be preferred? do we care if there are any with `""`? do we ever expect this to happen?